src/xcode/ENA/ENA/Source/Services/Risk/Provider/Model/RiskProvidingConfiguration.swift

```diff
@@ -31,6 +31,24 @@ struct RiskProvidingConfiguration {
 	var detectionMode: DetectionMode = DetectionMode.default
 }
 
+/// Either a concrete date or now
+enum NextExposureDetection: Equatable {
+	case date(Date)
+	case now
+
+	static func == (lhs: NextExposureDetection, rhs: NextExposureDetection) -> Bool {
+		switch (lhs, rhs) {
+		case (.now, .now):
+			return true
+		case let (.date(a), .date(b)):
+			// return true if dates are less than 30 seconds apart
```

Comment:
why 30 seconds? (having a comparison threshold makes perfect sense to me, I am just wondering if there is any further reasoning behind the value)

Reply:
-> marc told me its just an arbitrary value. fine for me!